src/main/java/edu/njit/jerse/ashe/services/SpeciminTool.java

```diff
@@ -69,13 +74,43 @@ public static String runSpeciminTool(String root, String targetFile, String targ
     }
 
     /**
-     * Creates a temporary directory for storing output from the Specimin tool.
+     * Deletes all files and subdirectories in the Specimin temporary directory. This method is specifically
```

Comment:
"in the Specimin temporary directory":  Is this true?  It seems to work on any given directory, and so the documentation and method name need to be updated.